src/api/migrations/20190708141337-create-stat.js

```diff
@@ -1,33 +1,30 @@
 export default {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('Reads', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      articleSlug: {
-        type: Sequelize.STRING,
-        references: { model: 'Articles', key: 'slug' }
-      },
-      userIp: {type: Sequelize.STRING},
-      userAgent: {type: Sequelize.STRING},
-      userId: {
-        type: Sequelize.INTEGER,
-        references: { model: 'Users', key: 'id' }
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      }
-    });
-  },
-  down: (queryInterface, Sequelize) => {
-    return queryInterface.dropTable('Reads');
-  }
-};
\ No newline at end of file
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Reads', {
+    id: {
+      allowNull: false,
```

Comment:
@nkpremices why do you have so many changes unrelated to your work? You're making changes in comments controller, tags controller etc

Reply:
@MemunaHaruna I had to create a route for a single route because I needed it in the email that I send when a comment is reported.

In the article controller, I noticed that @alainburindi added an if close which was not passing. Consequence, article tags were not being created. So I had to make them be created.  

In the password reset, there is something I had to modify to have the notifications template working with the template of password reset

Reply:
Okay